torchao/quantization/qat/fake_quantize_config.py

```diff
@@ -77,6 +77,24 @@ def __post_init__(self):
             )
 
 
+@dataclass
+class Int4WeightFBGEMMFakeQuantizeConfig(FakeQuantizeConfigBase):
```

Comment:
nit: maybe not mention FBGEMM, but `Preshuffled` instead to allign with the tensor name

Reply:
ok, should we use the following naming scheme?
```
Int4PreshuffledTensor -> Int4WeightPreshuffledFakeQuantizeConfig
Int4Tensor -> IntWeightFakeQuantizeConfig
```

Reply:
yeah, if the fake quant path uses the quant ops specific to Int4PreshuffledTensor (in from_hp), then I think we can just call it `Int4WeightPreshuffledFakeQuantizeConfig`